upkarlidder/coffee-donut-instructions
# Updating project

The project will be used at a conference. This PR updates the project to use latest TF and fixes breaking code due to this change.

- [x] updating the project python runtime. The old one is no longer available on IBM Cloud
- [x]  update TF version
- [x] fix breaking code with the TF version update
- [x] re-exported the project zip file

Steps to test the PR:
- sign into IBM Cloud
- ensure you have a COS service
- check out the branch
- import the assets/coffee-donut.zip project into Watson Studio
- run through the notebook

Signed-off-by: Upkar Lidder <ulidder@us.ibm.com>
